frontend/app/src/pages/main/managed-workers/create/components/create-worker-form.tsx

```diff
@@ -745,6 +755,301 @@ export default function CreateWorkerForm({
                 {memoryMbError && (
                   <div className="text-sm text-red-500">{memoryMbError}</div>
                 )}
+                <Label>Scaling Method</Label>
+                <Tabs
+                  defaultValue="static"
+                  value={hasAutoscaling ? 'autoscaling' : 'static'}
+                  onValueChange={(value) => {
+                    if (value === 'static') {
+                      setHasAutoscaling(false);
```

Comment:
re: above comment - then this can be something like this:

```typescript
value={scalingType}
onValueChange={(value) => {
  setScalingType(value)
  switch(value) {
    case "auto":
      setValue('runtimeConfig.numReplicas', undefined)
      ...
    case "static":
      setValue('runtimeConfig.numReplicas', 1);
      ...
  }
```